examples/schema.rs

```diff
@@ -18,29 +17,3 @@ fn main() {
     export_schema(&schema_for!(State), &out_dir);
     export_schema(&schema_for!(CountResponse), &out_dir);
 }
```

Comment:
You can add `remove_schemas` now (https://github.com/CosmWasm/cosmwasm/pull/282/files#diff-d9cf9a2c5387a424a227294fa17aac38)